.github/workflows/release.yml

```diff
@@ -0,0 +1,38 @@
+name: Kazoo Awesome Release
+
+on: [push]
+
+jobs:
+  build-and-release:
+    name: Build and release Kazoo to Pypi
+    runs-on: ubuntu-latest
+    steps:
+    - name: Handle the code
+      uses: actions/checkout@v2
+
+    - name: Set up Python 3.9
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.9
+
+    - name: Install pypa/build
+      run: >-
+        python -m
+        pip install
+        build
+        --user
+
+    - name: Build a binary wheel and a source tarball
+      run: >-
+        python -m
+        build
+        --sdist
+        --wheel
+        --outdir dist/
+        .
+
+    - name: Publish Kazoo to PyPI
+      if: startsWith(github.ref, 'refs/tags')
+      uses: pypa/gh-action-pypi-publish@master
+      with:
+        password: ${{ secrets.PYPI_API_TOKEN }}
```

Comment:
@bbangert I, unfortunately, don't have any access to the account used to publish to Pypi. Can you please add a secret called `PYPI_API_TOKEN` so that the CI will use to publish future versions of Kazoo? ([here the documentation](https://packaging.python.org/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/#saving-credentials-on-github) if needed)